src/app/charts/core/files/secret/tpl/arkcase-server.yaml

```diff
@@ -75,8 +75,17 @@ fileManagement:
 {{- end }}
 
 {{- if (eq "box" $content.dialect) }}
-box:
-  privateKeyPath: "${CONTENT_MAIN_PRIVATE_KEY_PATH:/app/home/.content-private-key}"
+  {{- $cmInfo := (.Files.Get "cminfo.yaml" | fromYaml) }}
+  {{- $privateKeyDict := dict "path" "/app/home/.content-private-key" }}
+
+  {{- range $key, $cm := $cmInfo }}
+    {{- if and (hasKey $cm "private-key") (hasKey (get $cm "private-key") "path") }}
+      {{- $_ := set $privateKeyDict "path" (get (get $cm "private-key") "path") }}
+    {{- end }}
+  {{- end }}
+
+  box:
+    privateKeyPath: "{{ $privateKeyDict.path }}"
```

Comment:
If you use the above one-liner, then this line becomes:

`privateKeyPath: {{ $privateKeyPath | quote }}`

Always use `quote` instead of quoting manually, since that function will escape any double-quotes in the value correctly.